bigquery_etl/parse_udf.py

```diff
@@ -14,20 +14,32 @@
 
 
 UDF_DIRS = ("udf", "udf_js")
+MOZFUN_DIR = ("mozfun",)
```

Comment:
Once we consolidate how UDFs are defined in mozfun and in udf/udf_js, this script can get simplified and refactored.